docs/reference-guide/deployment/kubernetes.md

```diff
@@ -42,35 +81,32 @@ spec:
               mountPath: /var/lib/memgraph
             - name: memgraph-log-storage
               mountPath: /var/log/memgraph
-            - name: memgraph-etc-config
-              mountPath: /etc/memgraph/memgraph.conf
-              subPath: memgraph.conf
-      volumes:
-        - name: memgraph-lib-storage
-          persistentVolumeClaim:
-            claimName: memgraph-lib-pv-claim
-        - name: memgraph-log-storage
-          persistentVolumeClaim:
-            claimName: memgraph-log-pv-claim
-        - name: memgraph-etc-config
-          configMap:
-            name: memgraph-config
 ---
-# Service
 apiVersion: v1
-kind: Service
+kind: PersistentVolumeClaim
 metadata:
-  name: memgraph-svc
-  labels:
-    app.kubernetes.io/name: memgraph
-    app.kubernetes.io/managed-by: Helm
+  name: memgraph-lib-storage
 spec:
-  type: ClusterIP
-  ports:
-    - port: 7687
-      targetPort: 7687
-      protocol: TCP
-      name: bolt
-  selector:
-    app.kubernetes.io/name: memgraph
-```
\ No newline at end of file
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 500Mi
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: memgraph-log-storage
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 500Mi
+
+```
+
+The above helm chart will spin up the Memgraph and expose it via **NodePort** service on port `7687` for communication via Bolt protocol. The helm chart also creates two **PersistentVolumeClaims** for the storage of the data directory and log directory. Since the Memgraph docker image has a root privilege on volumes data and log directories, it is necessary to set the `runAsUser` to `0` in the `securityContext` of the pod. This will override the memgraph user from the Docker image. Aldo is not ideal practice. Currently, the Memgraph must have root privileges on the volumes. 
+
+ The memgraph starts with the `--also-log-to-stderr=true` flag, meaning the logs will also be written to the standard error output. This is useful for getting logs via `kubectl logs` command.
+
```

Comment:
```suggestion
The above Helm chart spins up Memgraph and exposes it via **NodePort** service on port `7687` for communication via the Bolt protocol.

 The chart also creates two **PersistentVolumeClaims** for storing the data and log directories. Since Memgraph Docker image has root privileges on the data located on volumes and log directories, it is necessary to set the `runAsUser` to `0` in the `securityContext` section of the pod to override the `memgraph` user from the Docker image. Currently, Memgraph must have root privileges on the volumes. 

Memgraph starts with the `--also-log-to-stderr=true` flag, meaning the logs will also be written to the standard error output and you can access logs using the `kubectl logs` command.

```

Check if technically correct :)

AS Memgraph docker image has root user you don't have to set runAsUser? ASking for mysefl

Reply:
You need to, as we discussed on the call, BCS rights from docker images get mixed with root rights. Setting everything to the root avoids the issue.